modules/client/examples/05-encoders/03-freeze-permissions.ts

```diff
@@ -0,0 +1,38 @@
+/* MARKDOWN
```

Comment:
This permission no longer exists. The encoder should not exist either.

@josemarinas didn't we remove these encoder/decoder?